packages/quantic/quantic-examples-community/experiences/Quantic_Examples1/views/home.json

```diff
@@ -10,7 +10,7 @@
   }, {
     "components" : [ {
       "componentAttributes" : {
-        "richTextValue" : "<p><span style=\"font-size: 36px;\">Quantic Examples</span></p><p><br></p><p><span style=\"font-size: 24px;\">Typical Use Cases</span></p><p><br></p><ul><li><a href=\"full-search-example\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Full Search</a></li></ul><p><br></p><p><span style=\"font-size: 24px;\">Quantic Components</span></p><p><br></p><ul><li><a href=\"quantic-facet\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Facet</a></li><li><a href=\"quantic-facet-manager\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Facet Manager</a></li><li><a href=\"quantic-pager\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Pager</a></li><li><a href=\"quantic-placeholder\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Placeholder</a></li><li><a href=\"quantic-results-per-page\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Results Per Page</a></li><li><a href=\"quantic-sort\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Sort</a></li></ul>"
+        "richTextValue" : "<p><span style=\"font-size: 36px;\">Quantic Examples</span></p><p><br></p><p><span style=\"font-size: 24px;\">Typical Use Cases</span></p><p><br></p><ul><li><a href=\"full-search-example\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Full Search</a></li></ul><p><br></p><p><span style=\"font-size: 24px;\">Quantic Components</span></p><p><br></p><ul><li><a href=\"quantic-category-facet\" rel=\"noopener noreferrer\" target=\"_blank\" style=\"font-size: 16px;\">QuanticCategoryFacet</a></li><li><a href=\"quantic-facet\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Facet</a></li><li><a href=\"quantic-facet-manager\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Facet Manager</a></li><li><a href=\"quantic-pager\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Pager</a></li><li><a href=\"quantic-placeholder\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Placeholder</a></li><li><a href=\"quantic-results-per-page\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Results Per Page</a></li><li><a href=\"quantic-sort\" rel=\"noopener noreferrer\" target=\"_self\" style=\"font-size: 16px;\">Quantic Sort</a></li></ul>"
```

Comment:
I'd suggest to use `target="_self"` on the `QuanticCategoryFacet` link to open it in the current page like other links.